rest-api/src/main/java/com/cloudbees/workflow/flownode/FlowNodeListCacheAction.java

```diff
@@ -1,81 +0,0 @@
-/*
- * The MIT License
- *
- * Copyright (c) 2013-2016, CloudBees, Inc.
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
- */
-package com.cloudbees.workflow.flownode;
-
-import hudson.model.InvisibleAction;
-import hudson.model.Queue;
-import hudson.model.Run;
-import org.jenkinsci.plugins.workflow.graph.FlowNode;
-
-import java.util.List;
-import java.util.logging.Level;
-import java.util.logging.Logger;
-
-/**
- * Simple action for caching pre processed/filtered/sorted node lists.
- *
- * @author <a href="mailto:tom.fennelly@gmail.com">tom.fennelly@gmail.com</a>
- */
-public class FlowNodeListCacheAction extends InvisibleAction {
```

Comment:
@jglick I had tested with it and didn't get any notification; after rechecking the manage old data area, it shows nothing. 

Test process:

- Launch jenkins via docker image
- Install pipeline stage view, and restart after install (v1.0)
- Create a simple pipeline with stages and echo statements, and run it
- Refresh page a few times to ensure it gets hit and cached by pipeline stage view 
- Upload HPI for new pipeline stage view Rest API + UI, restart jenkins
- Look at 'manage old data'  -- "No old data was found."

Unless there is a necessary step here that is missing in testing, it looks like it just ignores the unrecognized action.